docs/advanced-usage.rst

```diff
@@ -144,66 +182,146 @@ index ``cid``.
     :scale: 70%
     :align: center
 
+In order to prepare a transfer transaction, alice needs to provide at least
+three things:
+
+1. ``inputs`` -- one or more conditions that will be fulfilled.
+2. ``asset`` -- the asset being transferred.
+3. ``owners_after`` -- one or more public keys representing the new owner(s).
+
+To construct the input:
+
+.. code-block:: python
+
+    cid = 0
+    condition = tx['transaction']['conditions'][cid]
+    input_ = {
+        'fulfillment': condition['condition']['details'],
+        'input': {
+            'cid': cid,
+            'txid': tx['id'],
+        },
+        'owners_before': condition['owners_after'],
+    }
+
+The asset, can be directly retrieved from the input ``tx``:
```

Comment:
I think it's good that we describe what's happening here, but ultimately I think we should allow the user to have some functionality that is similar to `Transaction.to_input` as it's quite painful to construct this payload by oneself (+ not everybody will always get these things right).